src/Components/ClusterHeader/ClusterHeader.js

```diff
@@ -36,6 +61,20 @@ export const ClusterHeader = ({ clusterId, clusterData }) => {
           </Title>
         )}
       </GridItem>
+      <GridItem span={4} id="cluster-header-dropdown">
+        <Dropdown
+          position="right"
+          onSelect={dropDownSelect}
+          autoFocus={false}
+          isOpen={isOpen}
+          toggle={
+            <DropdownToggle id="toggle-id-2" onToggle={dropDownToggle}>
```

Comment:
A minor suggestion: if it is a one-liner function, you can use an arrow function just right here without creating and referencing a new function.